### DISCUSSION TITLE
7월 19일 (월) 강좌 유튜브 Update 일정 문의

### DISCUSSION THREAD

**Discussion by SheungHoon-Lee** [fusiondevkr/pl900#26] [1 Q&A]
안녕하세요 이승훈입니다. 
Kick-Off 미팅 후, 숙제를 완료하고 기다리던 중, 개인사정으로 8월 19일 (월) 강의 참여를 못했습니다.
언제쯤 유튜브 강의를 볼 수 있을까요?

**Comment by justinyoo** [comment]
@SheungHoon-Lee 방금 자료를 업데이트했습니다! 👉 [바로가기](https://aka.ms/fdk/pl900/materials)

**Reply by sesme100** [reply]
퓨전 개발자 스터디 유투브는 사랑입니다!!
